dashboard/db/migrate/20191023192120_create_user_level_infos.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUserLevelInfos < ActiveRecord::Migration[5.0]
+  def change
+    create_table :user_level_infos do |t|
+      t.integer :time_spent, default: 0
+      t.bigint :user_level_id, unsigned: true
```

Comment:
Please try this instead:
```suggestion
      t.references :user_level, foreign_key: true
```
To do this, you will need to rollback the migration in its current state, change this code line, and then migrate forward again.

Reply:
This was failing yesterday because of the issue that is talked about here: https://github.com/rails/rails/issues/30806 . When I asked in infra channel they said it was okay to move forward without the foreign_key.

Reply:
got it. they would know best! sounds good.